pos_operating_unit/views/pos_views.xml

```diff
@@ -5,22 +5,19 @@
         <field name="model">pos.config</field>
         <field name="inherit_id" ref="point_of_sale.pos_config_view_form" />
         <field name="arch" type="xml">
-            <xpath expr="//field[@name='company_id']/../../.." position="after">
-                <div class="col-xs-12 col-lg-6 o_setting_box">
-                        <div class="o_setting_left_pane" />
-                        <div class="o_setting_right_pane">
-                            <div
-                            class="content-group"
-                            groups="operating_unit.group_manager_operating_unit"
-                        >
-                                <label for="operating_unit_ids" />
-                                <field
-                                name="operating_unit_ids"
-                                widget="many2many_tags"
-                            />
-                            </div>
-                        </div>
-                    </div>
+            <xpath expr="//setting[@id='other_devices']" position="before">
+                <setting
+                    id="operating_units_setting"
+                    string="Operating Units"
+                    help="Operating units used for this POS"
+                >
```

Comment:
@jdidderen-nsi Why remove groups?

Reply:
For no reason, a simple mistake when copying example for standard Odoo. 
I added the groups on every operating_unit_id(s) field to fit with other OU modules